book/src/attribs.md

```diff
@@ -0,0 +1,53 @@
+# Attributes for Crate Authors
+
+In some cases it is possible to extend Clippy coverage to 3rd party libraries.
+To do this, Clippy provides attributes that can be applied to items in the 3rd party crate.
+
+## `#[clippy::format_args]`
+
+_Available since Clippy v1.84_
+
+This attribute can be added to a macro that supports `format!`-like syntax.
+It tells Clippy that the macro is a formatting macro, and that the arguments to the macro
+should be linted as if they were arguments to `format!`. Any lint that would apply to a
+`format!` call will also apply to the macro call. The macro may have additional arguments
+before the format string, and these will be ignored.
```

Comment:
There are lints that apply to `format!` that wouldn't apply to macros marked with the attr, might want to list it as `format!`/`println!`/etc or similar

Reply:
e.g. `format_in_format_args`

Reply:
@Alexendoo thx, updated